### ISSUE TITLE
Multiple buttons are missing from the toolbar?

### ISSUE THREAD

**Issue by FabriceCastel** [blackears/cyclopsLevelBuilder#112]
I'm running version 1.0.1 (according to `plugin.cfg`)

The "delete" and "edit" buttons being omitted are what's causing an issue for me, as I'm trying to rotate a shape and take the intersection of that and another block..

Toolbar in the documentation screenshots:

<img width="744" alt="image" src="https://github.com/blackears/cyclopsLevelBuilder/assets/2886675/61e3e3ff-34f3-4a0d-b0aa-f08dfd4d9b27">

Toolbar in my editor:

<img width="698" alt="image" src="https://github.com/blackears/cyclopsLevelBuilder/assets/2886675/820394c6-6ca0-415a-90cc-01a15b6f394c">

Not sure if this is a case of me doing something wrong, the docs being out of date, or.. a different bug entirely?

I'm running godot on an M1 macbook pro, in case that's relevant. No errors/complaints in the debug output 🤔 not sure what's up.

**Comment by FabriceCastel** [comment]
I pulled `master` and "upgraded" the addon. Holy moly what a pain in the rear that was though. Class resolution errors up the wazoo, and restarting godot a bunch of times and even nuking the `.godot` dir didn't do it. I ended up having to go in and MANUALLY add the autoload command to my project config file because the GUI wouldn't let me (it complained about a conflict with an existing autoloaded script?).

Anyhow, have the greatest & latest now, will be playing with it some. Greatly appreciate your work on this plugin 🙏
